### DISCUSSION TITLE
Any plans to support PT-BR (Portuguese-Brazil)?

### DISCUSSION THREAD

**Discussion by KryTowskki** [bkaankose/Wino-Mail#233] [Q&A] [answered]
The app is great and has a ton of languages right now, but is lacking portuguese-brazil. I saw some reviews on MS Store also asking for this, and if you need help translating, i can help

**Comment by bkaankose** [comment, marked as answer]
Thank you for your contribution.

(Portuguese - Brazil) language is added to Crowdin project. Once translations reach up to %80 [on Crowdin](https://crowdin.com/project/wino-mail) I will add it to the next release.

**Reply by KryTowskki** [reply]
Thanks! Working on it.

**Reply by KryTowskki** [reply]
Hi, just updating you that i have reached 100% translated on Crowdin!

**Reply by bkaankose** [reply]
Thanks a lot. I will include the language to the next release in few days!
